src/main/java/blusunrize/immersiveengineering/common/blocks/IEStairsBlock.java

```diff
@@ -51,36 +55,16 @@ public IEStairsBlock setHasFlavour(boolean hasFlavour)
 		return this;
 	}
 
-	@Override
-	public float getExplosionResistance(Entity exploder)
-	{
-		return explosionResistance;
-	}
-
-	public IEStairsBlock setExplosionResistance(float explosionResistance)
-	{
-		this.explosionResistance = explosionResistance;
-		return this;
-	}
-
-	@Override
-	public boolean doesSideBlockRendering(BlockState state, IBlockAccess world, BlockPos pos, Direction face)
-	{
-		if(this.renderLayer!=BlockRenderLayer.SOLID)
-			return false;
-		return super.doesSideBlockRendering(state, world, pos, face);
-	}
-
-	public IEStairsBlock setRenderLayer(BlockRenderLayer layer)
-	{
-		this.renderLayer = layer;
-		return this;
-	}
-
+	//TODO: review: added flavour information here, so that `ItemBlockIEStairs` can be removed.
```

Comment:
1. You're right, that class can probably be removed now.
2. The implementation seems overly complicated. The method shouldn't be called for anything but the `this`-block, so it could be something like
```java
if (hasFlavour)
tooltip.add(new TranslationTextComponent(Lib.DESC_FLAVOUR+getRegistryName().getPath()));
```

Reply:
Aye, that was a bit overdone. Applied your suggested code.